src/index.js

```diff
@@ -0,0 +1,3 @@
+const userReady = confirm('Are you ready?');
```

Comment:
No, you should get from user some input(prompt) and transform it to boolean type